mac/build.sh

```diff
@@ -25,18 +25,32 @@ display_usage() {
     echo "  -clean          Removes all previously-existing build products for anything to be built before building."
     echo "  -test           Runs unit tests (not applicable to 'testapp' target)"
     echo "  -no-codesign    Disables code-signing for Keyman4MacIM, allowing it to be performed separately later"
```

Comment:
This comment could be better; that avoids the default codesigning. Using `-deploy local -no-codesign` actually does do signing, but not using the default certificate ID.